per-thread.md

```diff
@@ -0,0 +1,26 @@
+[✓] * Recorder.get_current() returns the current thread's recorder, if there is one, otherwise the
+      global recorder
+
+Global recording app:
+[✓] Flask
+[ ] Django
+* Is installed if should_enable("remote"), no other decisions should be made based on
+  should_enabled("remote")
+* Manages the state of remote recording separately from Recoder.get_current().enabled, so...
+* Doesn't make any decisions based on Recorder.get_current().enabled
+
```

Comment:
FYI the checkmarks were displayed horizontally rather than vertically.

![checkmarks](https://user-images.githubusercontent.com/111290954/200364494-371c115f-2735-438d-868c-52961a6a3480.png)

Reply:
Oops, these were just some notes for myself. I'll delete this file.

Reply:
Done.